pkg/volume/rbd/rbd.go

```diff
@@ -349,6 +380,9 @@ func (r *rbdVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 	if secretName == "" {
 		return nil, fmt.Errorf("missing user secret name")
 	}
+	if secretNamespace == "" {
+		secretNamespace = adminSecretNamespace
```

Comment:
This is incorrect. User secret namespace defaults to "", which uses the namespace of the pod/pvc

Reply:
done